crates/xilem_masonry/src/view/textbox.rs

```diff
@@ -61,6 +85,10 @@ impl<State, Action> MasonryView<State, Action> for Textbox {
     ) -> crate::ChangeFlags {
         let mut changeflags = ChangeFlags::UNCHANGED;
 
+        if self.contents != element.text().as_str() {
```

Comment:
This is very unidiomatic to Xilem. We should have a comment here mentioing that, at least

Reply:
Fair, it's kind of a band-aid. But it's needed to avoid calling set_text on every new keypress.